blog/2015-02-27-Docker-And-Wildfly-2.adoc

```diff
@@ -0,0 +1,115 @@
+= Docker and Wildfly - Part Two
+:page-layout: blog
+:page-author: rstryker
+:page-tags: [jbosstools, devstudio, server, docker, wildfly]
+
+If you've already link:2015-02-26-Getting-Started-With_Docker-And-Wildfly.html[gotten started] with trying out 
+Docker and JBoss Tools, and want to see what other options are available, this blog post will  
+explaini how to run Wildfly in a Docker container and configure it for management tasks and
+managed deployments. 
+
+=== Customizing your Dockerfile
+
+Since the default `jboss/wildfly` Docker configuration isn't good enough for us, we'll need
```

Comment:
lets make it more positive ;)

"...does not expose management port by default, we'll need "